client/devdocs/controller.js

```diff
@@ -1,43 +1,45 @@
 /**
  * External dependencies
  */
-var ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
-	qs = require( 'qs' ),
-	debounce = require( 'lodash/debounce' ),
-	page = require( 'page' ),
-	ReduxProvider = require( 'react-redux' ).Provider,
-	setSection = require( 'state/ui/actions' ).setSection,
-	EmptyContent = require( 'components/empty-content' );
+import ReactDom from 'react-dom';
+import React from 'react';
+import qs from 'qs';
+import debounce from 'lodash/debounce';
+import page from 'page';
+import { Provider } from 'react-redux';
+import { setSection } from 'state/ui/actions';
```

Comment:
Not new, but this one and `EmptyContent` should be in the internal dependencies block.